integration-tests/src/main/java/io/quarkiverse/minio/client/AsyncMinioController.java

```diff
@@ -0,0 +1,81 @@
+package io.quarkiverse.minio.client;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.concurrent.atomic.AtomicReference;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.QueryParam;
+
+import io.minio.BucketExistsArgs;
+import io.minio.MakeBucketArgs;
+import io.minio.MinioAsyncClient;
+import io.minio.PutObjectArgs;
+import io.minio.errors.InsufficientDataException;
+import io.minio.errors.InternalException;
+import io.minio.errors.XmlParserException;
+import io.smallrye.mutiny.Uni;
+import io.smallrye.mutiny.unchecked.Unchecked;
+
+@Path("/async-minio")
+public class AsyncMinioController {
+    // Over sizing chunks
+    private static final long PART_SIZE = 50 * 1024 * 1024;
+    public static final String BUCKET_NAME = "async-test";
+
+    @Inject
+    MinioAsyncClient minioAsyncClient;
+
+    @POST
+    public Uni<String> addObject(@QueryParam("name") String fileName) throws InsufficientDataException, IOException,
+            NoSuchAlgorithmException, InvalidKeyException, XmlParserException, InternalException {
+        final AtomicReference<InputStream> is = new AtomicReference<>();
```

Comment:
I wonder if we need such a complexity, we really just want to test that the async client is available, not that it works. That is minio-sdk's responsability.

Reply:
Alright, I'll simplify it